wgengine/router/router_darwin.go

```diff
@@ -10,55 +10,10 @@ import (
 	"tailscale.com/types/logger"
 )
 
-type darwinRouter struct {
```

Comment:
Does this break tailscaled on darwin now? People got it working recently.

Reply:
Connectivity works, just tested (it is pretty much the same as it was before, sans SetRoutesFunc). Nothing will happen to DNS, as `{up,down}DNS` below suggest. Perhaps this is the wrong behavior and DNS should be the same as on `freebsd`? My motivation for omitting it was primarily that it does pretty much nothing, as most things on macOS don't use `/etc/resolv.conf` AFAIK.